Update keycloak basic deployment guide (#239)
Apicurio/apicurio-registry-operator: docs/modules/ROOT/partials/proc-registry-security-keycloak.adoc

.Procedure
. In the OpenShift web console, click *Installed Operators* and *{keycloak} Operator*, and then the *Keycloak* tab.

. To create the {keycloak} instance we referr you to the Keycloak https://www.keycloak.org/operator/basic-deployment[documentation on how to create a basic deployment].

> . Once you have you {keycloak} instance available, you can now import the realm example we provide following the {keycloak} Operator https://www.keycloak.org/operator/realm-import[documentation].

Reviewer: ```suggestion
. When your {keycloak} instance is available, you can import the following realm example by using the steps in the https://www.keycloak.org/operator/realm-import[keycloak} Operator documentation]:
```
